simpeg/electromagnetics/static/spontaneous_potential/__init__.py

```diff
@@ -43,10 +47,19 @@
 
 """
 
-from .simulation import (
+import warnings
+
+warnings.warn(
+    "The spontaneous_potential module has been renamed to self_potential. Please use the self_potential module instead. The spontaneous_potential module will be removed in simpeg 0.23.",
```

Comment:
```suggestion
    "The spontaneous_potential module has been renamed to self_potential. Please use the self_potential module instead. The spontaneous_potential module will be removed in SimPEG 0.23.",
```

Reply:
Nice catch. I've updated this one already.